server/bin/config/custom-environment-variables.json

```diff
@@ -0,0 +1,3 @@
+{
+    "jwtprivatekey": "chefsmenujwtprivatekey"
```

Comment:
I think you don't want this file on the repo

Reply:
The value is the name of an env variable. This file basically just allows the config module to lookup the env variable so we can access non-sensitive config variables and sensitive environment variables through the same interface, i.e. config.get(propertyName).

Reply:
Oh ok, got it, I saw that and assumed it was the value of the key itself